presto-spi/src/main/java/com/facebook/presto/spi/security/Identity.java

```diff
@@ -29,18 +31,20 @@
     private final Optional<Principal> principal;
     private final Map<String, SelectedRole> roles;
     private final Map<String, String> extraCredentials;
+    private Map<String, String> extraInfo;
```

Comment:
Given the way it is being set, do you feel that just calling it `clientInformation` might be a better option?

Reply:
This is slightly more than the clientInfo we use at other places and in future we might add other info as well. So i went with extraInfo.  I am okay with either.